SparkPost/2web2ui
# SA-1117: Change UI to use rollups

### What Changed

- All requests outgoing to metrics are updated to use rollups

### How To Test

- On non hibana and hibana report pages, interact with the page to generate requests
- Verify any request going to /metrics/** contain the query param `rollup=true`. (note: `rollup`, not `rollups`)
- Verify date/time pickers are interactive

### To Do

- [ ] Update any tests
